docs/api-reference/indexing/partitioned.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Partitioned Index
+---
+
+<Note>
+  Creating BM25 indexes over partitioned tables is a ParadeDB enterprise
+  feature. [Contact us](mailto:sales@paradedb.com) for access.
+</Note>
+
+## Partitioned Index
+
+In Postgres, a partitioned index is an index created over a [partitioned table](https://www.postgresql.org/docs/current/ddl-partitioning.html).
+A BM25 index can be created over a partitioned table in the same way as a normal table.
+
+## Partial Index
```

Comment:
Is a partial index a type of partitioned index? If not, this feels confusing to me. Is it an enterprise feature too, or not?